src/shared/firewall-util-nft.c

```diff
@@ -815,6 +845,74 @@ static int fw_nftables_recreate_table(sd_netlink *nfnl, int af, sd_netlink_messa
         return 0;
 }
 
+static void nft_in6addr_to_range(const union in_addr_union *source, unsigned int prefixlen,
+                                 struct in6_addr *start, struct in6_addr *end)
+{
```

Comment:
nitpick: we put the { on the same line as the function declaration, i.e. at the end of the previous line, as per coding style

Reply:
nitpick: we like to prefix the name of return parameters with `ret_` to make clear they are return parameters, hence, please `start`→ `ret-start` + `end` → `ret_end`